src/main/java/com/minecolonies/blockout/controls/TextField.java

```diff
@@ -316,10 +316,10 @@ public void onFocus()
         cursorBlinkCounter = 0;
     }
 
-    @Override
     /**
      * Draw itself at positions mx and my.
      */
+    @Override
     protected void drawSelf(final int mx, final int my)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "drawSelf" is 21 which is greater than 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 24 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)